src/babel-register/register.js

```diff
@@ -0,0 +1,29 @@
+const { resolve } = require('path');
+
+// this must happen before `require('babel-register')` and can't be changed
+// once the module has been loaded
+if (!process.env.BABEL_CACHE_PATH) {
+  process.env.BABEL_CACHE_PATH = resolve(__dirname, '../../optimize/.babelcache.json');
+}
+
+// paths that babel-register should ignore
+const ignore = [
+  /[\\\/](node_modules|bower_components)[\\\/]/,
+];
+
+if (typeof BUILT_WITH_BABEL !== 'undefined') {
```

Comment:
Haha, yeah, fair 👍

Reply:
Actually, now that I think about it, I kind of like that it looks unnatural, since it totally is. But I'll switch over to process.env.BUILT_WITH_BABEL and leave a comment that the environment variable should never be set... idk...

Reply:
Maybe I can just replace `typeof BUILT_WITH_BABEL` with the string "boolean" or something? I just don't want people to get the idea that they need to set some environment variable, and I really don't want to read an environment variable here just because... or maybe `if (global.__BUILT_WITH_BABEL__)`

Reply:
Good idea with the `global`, definitely a better choice than my `process.env` idea :+1: